src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/ActionButtons.tsx

```diff
@@ -17,6 +17,7 @@ import DownloadStatus from '../Interfaces/DownloadStatus';
 
 interface ActionButtonsProps {
   isUserLoggedIn: boolean;
+  doesUerHasAccessToDownload: boolean;
```

Comment:
small typo
```suggestion
doesUserHaveAccessToDownload: boolean;
```
or
`userHasAccessToDownload: boolean;`

Reply:
nice catch, not sure why my spell checker didn't highlight that for me 🤦